Makefile

```diff
@@ -67,6 +67,47 @@ build-all:
 aro: check-release generate
 	go build -ldflags "-X github.com/Azure/ARO-RP/pkg/util/version.GitCommit=$(VERSION)" ./cmd/aro
 
+
+# Target to create docker secrets
+.PHONY: docker-secrets
+docker-secrets: aks.kubeconfig
+	docker secret rm --ignore aks.kubeconfig
+	docker secret create aks.kubeconfig ./aks.kubeconfig
+
+	docker secret rm --ignore proxy-client.key
+	docker secret create proxy-client.key ./secrets/proxy-client.key
+
+	docker secret rm --ignore proxy-client.crt
+	docker secret create proxy-client.crt ./secrets/proxy-client.crt
+
+	docker secret rm --ignore proxy.crt
+	docker secret create proxy.crt ./secrets/proxy.crt
```

Comment:
IIRC we aren't supposed to use docker command any more.

https://www.redhat.com/sysadmin/podman-kubernetes-secrets
podman also has secret command. Doesn't it work?

Reply:
We could move this to strictly say `podman` - it's been assumed for a while that all ARO devs are using `podman-docker` library to effectively `alias docker=podman` and not use the `docker` binary - `podman` is what I was testing with locally under the hood here.

I can update if we've decided to move away from the `docker` alias - but `docker` is ubiquitous throughout the Makefile today. If we want to do that, I think it should be done all at once and in a different PR maybe?

Reply:
FWIW +1 for being explicit with `podman` and +1 for a separate PR to convert the lot

Reply:
If we concern downstream impacts, we can do one by one starting with this PR.
IMHO I don't like the way to do all at once because we can't determine what breaks what.

And also `runlocal-portal` uses podman, so we're good to use podman here as well, I guess.

Reply:
I created a follow-up PR for this (and some other things I've noticed we need) https://github.com/Azure/ARO-RP/pull/3775